index.js

```diff
@@ -1,24 +1,79 @@
-function rot13(str) { // LBH QVQ VG!
-var shifted = 13;
```

Comment:
Always format your code.
Install something like https://packagecontrol.io/packages/HTML-CSS-JS%20Prettify for the first time